content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -124,7 +124,7 @@ When used alongside DVC, DVCLive can create _checkpoint_ signal files used by
 DVC <abbr>experiments<abbr>.
 
 This will save all the outputs (metrics, plots, models, etc.) associated to each
-[`step`](/doc/dvclive/api-reference/get_step).
+[`step`](/doc/dvclive/api-reference/live/get_step).
```

Comment:
Do you think we need to set a [redirect](https://github.com/iterative/dvc.org/blob/master/redirects-list.json) from the old location to the new one inside live/ ? In case people out there had previously linked from 3rd party pages and are now getting 404s.

Reply:
@daavoo if you see it fit please create the redirect. A single one for the whole section should be enough (the file uses regexps). Thanks